config/routes.php

```diff
@@ -200,9 +200,9 @@
         ['pass' => ['id', 'relation'], '_name' => 'modules:resourcesJson']
     );
     $routes->connect(
-        '/:object_type/view/:id/treeJson/:relation',
+        '/:object_type/view/:id/treeJson',
         ['controller' => 'Modules', 'action' => 'treeJson'],
-        ['pass' => ['id', 'relation'], '_name' => 'modules:treeJson']
+        ['pass' => ['id', 'root'], '_name' => 'modules:treeJson']
```

Comment:
we can remove `root' from `pass` array now since is not passed as argument to `treeJson` method anymore